tests/ui/test_home.py

```diff
@@ -21,3 +19,52 @@ def test_most_popular_extensions_are_sorted_by_users(
     extensions = page.most_popular.extensions
     sorted_by_users = sorted(extensions, key=lambda e: e.users, reverse=True)
     assert sorted_by_users == extensions
+
+
+@pytest.mark.smoke
+@pytest.mark.nondestructive
+def test_that_clicking_on_addon_name_loads_details_page(
+        my_base_url, selenium, initial_data):
+    page = Home(selenium, my_base_url).open()
+    page.click_on_first_addon()
+    page.wait_for_page
+    assert 'addon' in selenium.current_url
```

Comment:
So this is an example of a very vague test. We click an addon, wait for the URL to change, and then assert that 'addon' is in the URL. This isn't really checking anything useful other than we were able to click on an addon, but other tests will do that too. I would either replace this with a test that does more, or perform a check that's a better indication of the details page loading. I suspect other tests will cover this functionality better by doing something with the details page after clicking an add-on, but it's also possible those tests will load the details page directly (for performance reasons), which then makes it important to test clicking the add-on.